refine

```diff
@@ -585,6 +590,7 @@ run() {
         IS_JAR=`ls $REFINE_LIB_DIR | grep openrefine`
         if [ -z "$IS_JAR" ] ; then
             mvn_prepare
+            build_prepare
```

Comment:
I have added this to make sure we can run `./refine` just after cloning the repository (implicitly building before launching) as this is supported on Linux.